protocol/src/main/protobuf/Transactor.proto

```diff
@@ -14,14 +14,34 @@ message UpdateRequest {
     bytes chainCellCbor = 1;
 }
 
+message JournalStreamRequest {
+    MultihashReference lastJournalBlock = 1;
+}
+
+message ChainUpdateDescription {
+    MultihashReference canonicalRef = 1;
+    MultihashReference chainHeadRef = 2;
+}
+
+message JournalEvent {
+    oneof event {
+        MultihashReference canonicalInserted = 2;
+        ChainUpdateDescription chainUpdated = 3;
+        MultihashReference journalBlockPublished = 4;
+    }
+}
+
 service TransactorService {
     rpc InsertCanonical(InsertRequest) returns
         (MultihashReference);
 
     rpc UpdateChain(UpdateRequest) returns
-        (MultihashReference);
+        (ChainUpdateDescription);
```

Comment:
The client already has the canonical in order to do the update, so it doesn't need it.
What could be perhaps useful is the chainPrevious in the ChainEntry.